.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+python:
+  - "3.6"
+  
+install:
+  - pip install -r requirements.txt
+
+script: python -m pytest
```

Comment:
Why do you use "python -m pytest" instead of "python -m unittest"?

Reply:
Was recommended to me by Travis CI to use either pytest or tox. I chose pytest because it is compatible with unittest and is able to run all of the unittest module. Travis CI also already comes with it installed in their python env. I guess it's possible to run "python -m unittest" if you think it's better.